packages/google_maps_flutter/google_maps_flutter/lib/src/controller.dart

```diff
@@ -18,6 +18,14 @@ class GoogleMapController {
   /// The mapId for this controller
   final int mapId;
 
+  /// List of active stream subscriptions for map events.
+  ///
+  /// This list keeps track of all event subscriptions created for the map,
+  /// including camera movements, marker interactions, and other map events.
+  /// These subscriptions are properly disposed when the controller is disposed.
+  final List<StreamSubscription<dynamic>> _streamSubscriptionList =
```

Comment:
Generally variable names shouldn't include their types unless they communicate some important information. This can just be called `_streamSubscriptions`.